projects/novo-elements/src/elements/common/typography/label/label.component.ts

```diff
@@ -21,4 +23,13 @@ import { NovoBaseTextElement } from '../base/base-text.component';
     class: 'novo-label',
   },
 })
-export class NovoLabel extends NovoBaseTextElement {}
+export class NovoLabel extends NovoBaseTextElement {
+  @HostBinding('attr.id')
+  public id: string;
+
+  inputId = input(null, { alias: 'id' });
+
+  ngOnInit() {
```

Comment:
it's generally good practice to implement the OnInit interface on the class for type checking, but we also might be able to eliminate this entirely by just making the `id` property a computed signal and then referencing it on a new `host` property in the component decorator.